entrypoint.sh

```diff
@@ -7,8 +7,8 @@ function run_scripts () {
 	SCRIPTS=$(find "$SCRIPTS_DIR" -type f -uid 0 -executable -regex "$SCRIPT_FILES_PATTERN" | sort)
 	if [ -n "$SCRIPTS" ] ; then
 		echo "=>> $1-scripts:"
-	    for script in $SCRIPTS ; do
-	        echo "=> $script"
+		for script in $SCRIPTS ; do
+			echo "=> $script"
```

Comment:
[this](https://github.com/funkyfuture/docker-drupal/blob/master/entrypoint.sh#L8)? it's a diff against this samos123/docker-drupal:master, I'm not sure what to tell you about your master.